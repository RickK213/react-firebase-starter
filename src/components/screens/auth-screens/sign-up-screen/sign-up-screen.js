import React from 'react';
import { SignUpForm } from './sign-up-form/sign-up-form';

export const SignUpScreen = () => {
  return (
    <div>
      <h2>Sign Up</h2>
      <SignUpForm />
    </div>
  );
};
