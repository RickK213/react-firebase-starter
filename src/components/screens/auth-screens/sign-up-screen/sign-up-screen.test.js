/* global React */
import ShallowRenderer from 'react-test-renderer/shallow';
import { createMemoryHistory } from 'history';
import { SignUpScreen } from './sign-up-screen';

describe('<SignUpScreen />', () => {
  const renderer = new ShallowRenderer();
  const history = createMemoryHistory({ keyLength: 0 });

  it('should render correctly', () => {
    renderer.render(<SignUpScreen history={history} />);
    const component = renderer.getRenderOutput();
    expect(component).toMatchSnapshot();
  });
});
