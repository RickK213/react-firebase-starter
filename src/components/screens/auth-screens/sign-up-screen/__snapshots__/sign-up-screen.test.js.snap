// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpScreen /> should render correctly 1`] = `
<div>
  <h2>
    Sign Up
  </h2>
  <withRouter() />
</div>
`;
